### ISSUE TITLE
Blogposts on members

### ISSUE THREAD

**Issue by adina** [swcarpentry/communications#16]
Reference https://github.com/adina/communications/issues/8 (@adina)

**Comment by amyrbrown** [comment]
So, say, a bi-weekly or monthly profile? Any particular information that should be covered in the profiles?

**Comment by jiffyclub** [comment]
@adina I think I owe you a post, pester me if I don't get it to you soon.

**Comment by adina** [comment]
@amyrbrown See https://github.com/swcarpentry/communications/blob/master/member-blogs/list-of-questions.md.  I am collecting blogposts from @jiffyclub and @katyhuff right now but perhaps you can help get these posted and then choose a couple other members to continue the blogging?

**Comment by katyhuff** [comment]
Hi @adina ! Check out #22 .

**Comment by adina** [comment]
Assigned to @amybrown.  Please let me know if you need help!

**Comment by iglpdc** [comment]
So, are these posts to have handy in case we need to publish them or just posts that we are going to publish on a regular basis in our blog? If we are going for the second, maybe we just need to keep here the list of questions, and not the blog post themselves (they should go to the site repo).

@adina could you write a small README in the member-blogs folder clarifying this?

**Comment by adina** [comment]
I was thinking of storing blogposts and releasing them every couple weeks?  e.g., collecting, storing, and then "limited releasing" so we have a continuous supply @amyrbrown.

**Comment by amyrbrown** [comment]
@adina, that sounds good. I'll work on the first couple and hopefully have them ready to go by the end of the weekend.

**Comment by amyrbrown** [comment]
Okay, I did a blog post for Katy's questions, but it was painful and I'm sure there's a better way.

Firstly, I didn't have direct access to Katy's answers through the pull request, so I just copied and pasted. I should probably have forked from Katy's, or maybe accepted the PR in the `communications` repo and then pulled it to my clone? Wait, I don't have write access to the `communications` repo.

Second, it looks like blog posts need to be written in HTML, but the answers are Markdown. Is there a nice way to get from MD to HTML, or is it actually okay to write blog posts in Markdown? (Actually it looks like Marked has a "save as HTML option, so maybe that will work okay?)

And finally, I couldn't test the blog post before I PR'd it because bitrot has gotten to my Jekyll or Ruby installs and I ran out of time. I'll try and troubleshoot that before I do the next one.

**Comment by iglpdc** [comment]
Oh, I'm sorry to hear it was painful. I think it would be easier to do this in the site repo. A possible workflow would be:

- we have a "profile-core" branch in the site repo, with just the list of questions and the YAML header, and probably README with a better version of these instructions,
- the person to be profiled checks out this branch and makes a new branch out it, say "profile-katy",
- when they are done, they submit a PR from that branch against the site repo,
- you checkout their branch with the PR (which now has their answers) and write the actual blog post,
- other people can review your post,
- when you're done you merge their PR (which now has also your contributions).

Sounds good? If this makes sense, I could try to write the instructions with the git-fu needed (it would be good practice for me, as long as someone review it later).

And, yes, you can use Marked to export markdown to html and then paste in the html file.

**Comment by adina** [comment]
@amyrbrown  Is it okay if I leave this up to you for rolling out?  I would suggest releasing one at a time every other week and in the mean time reach out to other members for similar blog posts -- that way we have a good "rotation" in place.

**Comment by amyrbrown** [comment]
@adina Yeah, I will handle it. I'd like to do some profiles on some newer and perhaps less-vocal members -- any suggestions? In the meantime I'll get Matt's lined up and think about who else to approach.

**Comment by amyrbrown** [comment]
@iglpdc That sounds like an excellent workflow. Thanks for coming up with it. Let me know if there's anything I can do to help implement it. (And sorry for the slow response -- it's March Break here so the children are home all day.)

**Comment by weaverbel** [comment]
I have started a Google doc on suggested ideas for the blog. Happy to have other contributors for ideas/contributors
https://docs.google.com/spreadsheets/d/1_Dk5WnJ7gCYOBEYu8TA3AfKgcg0CQZ8pyAXwezN3sbU/edit#gid=0 - just ping me
